sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -164,11 +164,15 @@ private void apiConnectionLogic(HttpServletRequest request, HttpServletResponse
 			clientRequest.addHeader(h.getName(), h.getValue());
 		}
 		BufferedReader rd = null;
+		long startTime = System.currentTimeMillis();
 		try {
 			rd = new BufferedReader(new InputStreamReader(client.execute(clientRequest).getEntity().getContent()));
 		} catch (IOException e) {
 			M_log.error("Canvas API call did not happen",e);
 		}
+		long stopTime = System.currentTimeMillis();
```

Comment:
This will not print the elapsed time if there is an exception that isn't an IOException.  This probably is fine in this case.